backend/src/main/java/ca/bc/gov/hlth/mohums/controller/ClientsController.java

```diff
@@ -1,34 +1,31 @@
 package ca.bc.gov.hlth.mohums.controller;
 
 import ca.bc.gov.hlth.mohums.util.AuthorizedClientsParser;
-import ca.bc.gov.hlth.mohums.webclient.WebClientService;
+import ca.bc.gov.hlth.mohums.webclient.KeycloakApiService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Is there a reason you changed the import to this from importing the explicit annotations you needed?

Reply:
Didn't do that on purpose - just realized that the keyboard shortcut I use for deleting unused imports, is 'optimizing' the imports as well - and grouped those 5 explicit into  `import org.springframework.web.bind.annotation.*;`
Will roll back to explicit ones.